Sources/KituraNIO/ClientRequest.swift

```diff
@@ -82,6 +82,8 @@ public class ClientRequest {
     /// The path (uri) related to the request, starting from / and including query parameters
     private var path = ""
 
+    let waitSemaphore = DispatchSemaphore(value: 0)
```

Comment:
@saiHemak Can we please have a comment to document the objective of this DispatchSemaphore?